src/robinhood.py

```diff
@@ -71,25 +69,43 @@ def get_ticker(cls, ticker):
         Robinhood.check_login_time()
 
         fundamentals = r.get_fundamentals(ticker)
+        instrument_data = r.find_instrument_data(ticker)
+
+        # Detect invalid index
         if fundamentals[0] == None:
             return False
 
         ticker_data = {}
+        ticker_data["market"] = instrument_data[0]["market"].strip("https://api.robinhood.com/markets/")
         ticker_data["current"] = round(float(r.stocks.get_latest_price(ticker)[0]),2)
         ticker_data["volume"] = round(float(fundamentals[0]["volume"]),2)
         ticker_data["avg_volume"] = round(float(fundamentals[0]["average_volume"]), 2)
         ticker_data["market_cap"] = round(float(fundamentals[0]["market_cap"]), 2)
-        ticker_data["market_status"] = 0
-        ticker_data["name"] = r.stocks.find_instrument_data(ticker)[0]['simple_name']
-        ticker_data['points_change'] = 0
+        ticker_data["name"] = r.stocks.find_instrument_data(ticker)[0]["simple_name"]
         ticker_data["range"] = {
             "open": round(float(fundamentals[0]["open"]),2),
             "high": round(float(fundamentals[0]["high"]),2),
             "low": round(float(fundamentals[0]["low"]),2),
             "date": fundamentals[0]["market_date"]
         }
-
         ticker_data["symbol"] = fundamentals[0]["symbol"]
         ticker_data["timestamp"] = (datetime.utcnow()).strftime("%H:%M:%S")
 
+        # Calculate points and percent change
+        price_change = {}
+        price_change["points"] = round(float(ticker_data["current"] - ticker_data["range"]["open"]),2)
+        price_change["percent"] = round(float(price_change["points"] / ticker_data["range"]["open"] * 100),2)
```

Comment:
Apparently the way we calculate price change for points and percent is wrong.

According to this [article](https://pocketsense.com/calculate-anexpected-growth-rate-using-constant-growth-3970.html):

> Subtract the initial share price of the stock from the ending share price of the stock to find the change in price. For example, if the stock started the day at $59.90 and ended the day at $59.60, subtract $59.90 from $59.60 to get a change of -30 cents, meaning you had a loss of 30 cents.
>
> Divide the change in the stock price by the initial price per share. In this example, divide the loss of 30 cents by the initial stock price of $59.90 to get about 0.005.
>
> Move the decimal place in 0.005 two places to the right to find the stock lost 0.5 percent of its value for the day.